src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -505,14 +505,14 @@ std::string postQueryContext
   //
   qcrsP->errorCode.fill(SccOk);
 
-  TIMED_MONGO(ciP->httpStatusCode = mongoQueryContext(qcrP,
-                                                      qcrsP,
-                                                      ciP->tenant,
-                                                      ciP->servicePathV,
-                                                      ciP->uriParam,
-                                                      ciP->uriParamOptions,
-                                                      countP,
-                                                      ciP->apiVersion));
+    TIMED_MONGO(ciP->httpStatusCode = mongoQueryContext(qcrP,
+                                                        qcrsP,
+                                                        ciP->tenant,
+                                                        ciP->servicePathV,
+                                                        ciP->uriParam,
+                                                        ciP->uriParamOptions,
+                                                        countP,
+                                                        ciP->apiVersion));
```

Comment:
Please avoid breaking existing indentation

```suggestion
  TIMED_MONGO(ciP->httpStatusCode = mongoQueryContext(qcrP,
                                                      qcrsP,
                                                      ciP->tenant,
                                                      ciP->servicePathV,
                                                      ciP->uriParam,
                                                      ciP->uriParamOptions,
                                                      countP,
                                                      ciP->apiVersion));
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/397ae9e496a9ce542907b10016aa464b69ba71a3